### ISSUE TITLE
We currently do not inject our entry code into single-file anonymous entries error

### ISSUE THREAD

**Issue by dehghani-mehdi** [smooth-code/error-overlay-webpack-plugin#37]
Hi,

Using `"error-overlay-webpack-plugin": "^0.2.0"` and `"webpack": "^4.30.0"`, I got this error:

> Error: We currently do not inject our entry code into single-file anonymous entries.
Please use a multi-main (array) or object-form `entry` setting for now.
    at adjustEntry (PATH\node_modules\error-overlay-webpack-plugin\lib\index.js:56:11)

After removing `error-overlay-webpack-plugin` everything works fine.

**Comment by vitormv** [comment]
Having same error here

**Comment by undermuz** [comment]
Have same error

**Comment by neoziro** [comment]
Hello, OK it is a bug. Could someone submit a PR to fix it?

**Comment by YaoKaiLun** [comment]
It's ok in v0.1.6 but not in v0.1.7.

**Comment by YaoKaiLun** [comment]
Change "entry: './static/src/index.js'" to "entry: ['./static/src/index.js']"

**Comment by neoziro** [comment]
Yeah you have to change entry I think, just that.

**Comment by dehghani-mehdi** [comment]
@neoziro if you can't fix the issue (because of time limit or whatever), at least don't close the issue with simple answer that don't even fix the issue (I guess you are not sure either as you mentioned __I think__ in your answer). obviously I can not change my webpack config just because your plugin likes it that way.

Please don't contribute to open source, but if you do be professional at it. I'll remove your plugin, simply because I can not write my webpack config based on your buggy code.

**Comment by neoziro** [comment]
Hello @dehghani-mehdi, if you are not happy feel free to fix this plugin with a PR or simply don't use it.

**Comment by dehghani-mehdi** [comment]
@neoziro, Hi,don't worry, I don't use your plugin anymore, I mentioned that in my prev comment.

**Comment by neoziro** [comment]
I don’t worry 🤗

**Comment by dehghani-mehdi** [comment]
Good for you ;)

**Comment by dehghani-mehdi** [comment]
To anyone who like to have error overlay without `error-overlay-webpack-plugin`, the solution is very simple, just run `npm i react-dev-utils -D` and change the `entry` option of your `webpack.config.js` to: `[require.resolve('react-dev-utils/webpackHotDevClient'), './index.jsx']`, I'm using this method for months.

**Comment by neoziro** [comment]
@dehghani-mehdi feel free to submit a PR with an upgraded version of `react-dev-utils`. I see that there is a new version but I don't have enough time to test if everything work on `react-dev-utils` v9.

**Comment by dehghani-mehdi** [comment]
@neoziro I cloned `error-overlay-webpack-plugin`, without any change the provided example works fine (_of course I changed `const ErrorOverlayPlugin = require('../lib')` to `const ErrorOverlayPlugin = require('../src')`_). also I updated `wabpack` packages and again everything works fine. I think I have to test the code with one of my projects. If I do (_and could fix the problem_), I'll post the result here or submit PR for it.
